generator/src/main/java/org/corfudb/generator/profiler/profiler.py

```diff
@@ -0,0 +1,506 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+input_path = "/Users/vshriya/Desktop/current/CorfuDB/test/client-1529017385323.log"
+output_path = "results/shriya/"
+points = []  # all data points, aka each entry in log files
+all_ops = []  # all operations mentioned in log files
+
+
+### Helper Functions
+def setup():
+    global points, all_ops
+
+    # open file and populate global vars
+    f = open(input_path)
+    for point in f.readlines():
+        points.append(point)
+        if get_event_name(point) not in all_ops:
+            all_ops.append(get_event_name(point))
+
+    # sort points by start time
+    points.sort(key=lambda x: get_time_stamp(x))
+
+
+def get_time_stamp(entry):
+    return int(entry.split()[0])
+
+
+def get_thread_ID(entry):
+    return int(entry.split()[1])
```

Comment:
Can you map these indices to constant names. For example, entry.split()[thread_id_ind], where thread_id_ind = 1